packages/oci_cli.rb

```diff
@@ -0,0 +1,36 @@
+require 'package'
+
+class Oci_cli < Package
```

Comment:
Please keep the old name `oci`.  No reason to rename to make the name longer and prevent updates for users that have installed with the original name.

Reply:
Oci is ambiguous. Oci could be referring to the OCI Python SDK (found at py3_oci) or the OCI CLI (found at oci_cli)

Reply:
Also I don't think anyone is using OCI anyway as the binaries are broken on all arches and there are no bug reports.